clicommand/agent_start.go

```diff
@@ -939,6 +948,23 @@ var AgentStartCommand = cli.Command{
 			l.Info("Agents will disconnect after %d seconds of inactivity", agentConf.DisconnectAfterIdleTimeout)
 		}
 
+		// Show allowed environment variables if it is set
+		if len(agentConf.AllowedEnvironmentVariables) > 0 {
+			l.Info("Environment variable allow list is enabled. Only allow-listed environment variables can be set.")
```

Comment:
```suggestion
```

none of the other config options emit a log when they get turned on, i think it's safe to get rid of this one. we can leave the debug logs though, it's okay to be a bit verbose with those